frontend/pages/books/[id].js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useRouter } from "next/router";
+import {
+  Breadcrumbs,
+  EmptyState,
+  Flex,
+  Layout,
+  Section,
+  Text,
+} from "../../components";
+import { fontWeight } from "../../theme";
+import { useGetBook } from "../../api/books";
+
+const BooksDetailPage = () => {
+  const router = useRouter();
+  const { book, bookLoading, bookError } = useGetBook(router.query.id);
+  return (
+    <Layout>
+      {bookLoading || bookError || !Object.keys(book).length ? (
```

Comment:
I think there is a better way we can handle this, this logic is confusing and not something we want to see all over our JSX. 

Remember to follow react best practices of keeping JSX lean.